keps/sig-auth/20190607-certificates-api.md

```diff
@@ -132,6 +134,105 @@ A typical successful issuance proceeds as follows.
    the `.Status.Certificate` field.
 1. The requestor observes the update, and stores the certificate locally.
 
+### Signers
+
+CSRs have a `signerName` field which is used to specify which signer the CSR creator wants to sign the certificate.
+To support migration from v1beta1 to v1, this required field will be defaulted in v1beta1 (optional in openapi), but
+not defaulted and required in v1 :
+ 1. If it's a kubelet client certificate, it is assigned "kubernetes.io/kubelet-client".
```

Comment:
nit: I think this should be `kube-apiserver-client-kubelet` now?

Reply:
fixed in https://github.com/kubernetes/enhancements/pull/1485/commits